www_src/pages/make/make.jsx

```diff
@@ -79,10 +80,9 @@ var Make = React.createClass({
       if (!body || !body.project) {
         return this.onEmpty();
       }
-      if (window.Android) {
-        window.Android.trackEvent('Make', 'Create a Project', 'New Project Started');
-        window.Android.setView('/users/' + user.id + '/projects/' + body.project.id);
-      }
+      
+      platform.trackEvent('Make', 'Create a Project', 'New Project Started');
```

Comment:
Using the platform polyfill for the entire "Make" activity including GA tracking, view transitions, and user session reset.